src/images/zookeeper.rs

```diff
@@ -0,0 +1,88 @@
+use crate::core::Port;
+use crate::{Container, Docker, Image, WaitForMessage};
+use std::collections::HashMap;
+
+const CONTAINER_IDENTIFIER: &str = "bitnami/zookeeper";
```

Comment:
Any reason we can't use the official zookeeper image? https://hub.docker.com/_/zookeeper